docs/references/protocol/transactions/types/ammbid.md

```diff
@@ -11,7 +11,7 @@ labels:
 
 _(Requires the [AMM amendment][])_
 
-Bid on an [Automated Market Maker](../../../../concepts/tokens/decentralized-exchange/automated-market-makers.md)'s (AMM's) auction slot. If you win, you can trade against the AMM at a discounted fee until you are outbid or 24 hours have passed. If you are outbid before 24 hours have passed, you are refunded part of the cost of your bid based on how much time remains.
+Bid on an [Automated Market Maker](../../../../concepts/tokens/decentralized-exchange/automated-market-makers.md)'s (AMM's) auction slot. If you win, you can trade against the AMM at a discounted fee until you are outbid or 24 hours have passed. If you are outbid before 24 hours have passed, you are refunded part of the cost of your bid based on how much time remains. A Bid on AMM pool without trading fee set may result in sucessful transaction but will have no effect.
```

Comment:
To my understanding:

- A pool can have a fee of 0 but it cannot have a fee not set.
- If you win the auction slot, you have access to the discounted fee for 24 hours. This is true regardless of whether the existing fee is 0 or a larger number. Of course, 10% of 0 is still 0, so it's not a useful discount during that time.
- The trading fee can change via AMM voting during the time you have the auction slot. So even if the trading fee is 0 when you first get the auction slot, the trading fee can go up later and you'll still be entitled to trading at 10% of the normal fee.

So it's not precisely right to say the transaction will have no effect.

Reply:
I suggest rephrasing to:
"A Bid on AMM pool with trading zero fee may result in successful transaction but will have no effect."